src/gridstack-engine.ts

```diff
@@ -31,6 +31,7 @@ export class GridStackEngine {
   public addedNodes: GridStackNode[] = [];
   public removedNodes: GridStackNode[] = [];
   public batchMode: boolean;
+  public defaultColumn = 12;
```

Comment:
I think instead of having a 'hard coded' max layout of 12, we can just use the largest size in the layout as the current max. will have to check code again...